docs/concepts.rst

```diff
@@ -602,6 +611,16 @@ as such. The third call uses the ``default_var`` parameter with the value
 isn't defined. The get function will throw a ``KeyError`` if the variable
 doesn't exist and no default is provided.
 
+You can use the ``default_var`` argument to make sure of not overwriting
+an exsisting Variable key :
+
+.. code:: python
+
+    from airflow.models import Variable
+    variable = Variable.get("foo", default_var=None)
+    if variable is None:
+        Variable.set("foo", 5)
+
```

Comment:
This is not needed and does not provide any new information that is not covered in the above examples